src/main/java/me/qoomon/gradle/gitversioning/GitVersioningPluginExtension.java

```diff
@@ -554,6 +553,7 @@ private Map<String, Supplier<String>> generateGlobalFormatPlaceholderMap(GitSitu
             placeholderMap.put("describe.tag." + groupName, groupValue);
             placeholderMap.put("describe.tag." + groupName + ".slug", Lazy.by(() -> slugify(groupValue.get())));
         }
+        final Lazy<Integer> distance = Lazy.by(() -> description.get().getDistance());
```

Comment:
please rename variable to `descriptionDistance`and move this statement next to where it is used (line 590)

Reply:
Done - see latest commit